examples/dynamic-openapi-joi.js

```diff
@@ -0,0 +1,67 @@
+'use strict'
+
+const fastify = require('fastify')()
+const joiToJson = require('joi-to-json')
+const Joi = require('joi')
+
+fastify.register(require('../index'), {
+  openapi: {
+    info: {
+      title: 'Test swagger',
+      description: 'testing the fastify swagger api',
+      version: '0.1.0'
+    },
+    servers: [{
+      url: 'http://localhost'
+    }],
+    components: {
+      securitySchemes: {
+        apiKey: {
+          type: 'apiKey',
+          name: 'apiKey',
+          in: 'header'
+        }
+      }
+    }
+  },
+  exposeRoute: true,
+  transform: schema => {
+    const {
+      params = undefined,
+      body = undefined,
+      querystring = undefined,
+      ...others
+    } = schema
+    const transformed = { ...others }
+    if (params) transformed.params = joiToJson(params)
```

Comment:
use `convert` here to match with the docs.
```suggestion
    if (params) transformed.params = convert(params)
```